apache/trafficcontrol
# Deprecate users/{userId}/deliveryServices

## What does this PR (Pull Request) do?


- [X] This PR fixes #4402, deprecating `users/{id}/deliveryservices` and adding the `accessibleTo` query parameter to `deliveryservices` 


## Which Traffic Control components are affected by this PR?


- Traffic Control Client 
- Traffic Ops
- Traffic Portal

## What is the best way to verify this PR?
Ensure docs build, TO tests pass, TP tests pass.

For TO:
- Verify `1.1/users/:id/deliveryservices` returns a deprecation notice (This endpoint does not work see #3754)
- Verify `2.0/users/:id/deliveryservices` doesn't exist
- Verify `2.0/deliveryservices?accessibleTo={tenantId}` works and returns all delivery services that have this tenant, or any delivery services that are children of the given tenant.

For TP:
- Verify when clicking the `View Delivery Services` button on the user editting screen (`users/{userId}`), it takes the user to `tenant/{tenantId}/deliveryservices`
- Verify that `tenant/{tenantId}/deliveryservices` returns the delivery services referencing this tenant, **and** this tenants children (previously does not show children).





## The following criteria are ALL met by this PR




## Additional Information
